.github/workflows/compatibility-matrix-testing.yml

```
@@ -107,28 +142,100 @@ jobs:
     strategy:
       fail-fast: false
       matrix: ${{ fromJson(inputs.CMT_MATRIX) }}
-    secrets: inherit
+    secrets:
+      MM_MOBILE_E2E_ADMIN_EMAIL: ${{ secrets.MM_MOBILE_E2E_ADMIN_EMAIL }}
```

actionlint (1.7.12)

[error] secret "MM_MOBILE_E2E_ADMIN_EMAIL" is not defined in "./.github/workflows/e2e-ios-template.yml" reusable workflow. no secret is defined

(workflow-call)